### PR TITLE
Replace mentions of Discourse with Github Discussions

### PR DESCRIPTION
**Pull request by StanczakDominik**
- [ ] I have added a changelog entry for this pull request.

- [ ] If adding new functionality, I have added tests and
      docstrings.

- [ ] I have fixed any newly failing tests.

### CODE REVIEW

File: docs/COMMUNICATION.rst [hunk 1]
```diff
@@ -6,19 +6,18 @@ Feedback and Communication
...
+get a response, for PlasmaPy is our `Matrix room
+<https://app.element.io/#/room/#plasmapy:openastronomy.org>`__. It is is
+bridged (mirrored) to `Gitter <https://gitter.im/PlasmaPy/Lobby>`__.
 
-If you have any questions, the quickest way to get a response is to ask
-on our `Matrix <https://app.element.io/#/room/#plasmapy:matrix.org>`__/ channel.
-
-`Discourse group <https://plasmapy.discourse.group>`__
+`GitHub Discussions <https://github.com/PlasmaPy/PlasmaPy/discussions>`__
 ------------------------------------------------------
```
Comment (rocco8773) [inline docs/COMMUNICATION.rst:15]:
```suggestion
`GitHub Discussions <https://github.com/PlasmaPy/PlasmaPy/discussions>`__
-------------------------------------------------------------------------
```

This is causing the doc build failure.

File: docs/COMMUNICATION.rst [hunk 2]
```diff
@@ -6,19 +6,18 @@ Feedback and Communication
 `Matrix chat <https://app.element.io/#/room/#plasmapy:matrix.org>`__
 --------------------------------------------------------------------
 
-The primary communication channel for PlasmaPy is our `Matrix room
-<https://app.element.io/#/room/#plasmapy:openastronomy.org>`__ (which
-is bridged to `Gitter <https://gitter.im/PlasmaPy/Lobby>`__).
+The primary communication channel, and the quickest way to ask a question and
+get a response, for PlasmaPy is our `Matrix room
```
Comment (rocco8773) [inline docs/COMMUNICATION.rst:10]:
```suggestion
The primary communication channel for PlasmaPy, and the quickest way to ask
a question and get a response, is our `Matrix room
```

File: docs/COMMUNICATION.rst [hunk 3]
```diff
@@ -6,19 +6,18 @@ Feedback and Communication
...
+<https://app.element.io/#/room/#plasmapy:openastronomy.org>`__. It is is
+bridged (mirrored) to `Gitter <https://gitter.im/PlasmaPy/Lobby>`__.
```
Comment (rocco8773) [inline docs/COMMUNICATION.rst:12]:
```suggestion
<https://app.element.io/#/room/#plasmapy:openastronomy.org>`__, which is also
bridged (mirrored) to `Gitter <https://gitter.im/PlasmaPy/Lobby>`__.
```

File: docs/COMMUNICATION.rst [hunk 4]
```diff
@@ -6,19 +6,18 @@ Feedback and Communication
...
-
-`Discourse group <https://plasmapy.discourse.group>`__
+`GitHub Discussions <https://github.com/PlasmaPy/PlasmaPy/discussions>`__
 ------------------------------------------------------
 
-PlasmaPy's `Discourse group <https://plasmapy.discourse.group>`__ is a
-great place to suggest ideas, bring up discussion topics, and ask
-questions in threaded public discussions.
+PlasmaPy's `GitHub Discussions page
+<https://github.com/PlasmaPy/PlasmaPy/discussions`__ is a great place to
```
Comment (rocco8773) [inline docs/COMMUNICATION.rst:18]:
```suggestion
PlasmaPy's `GitHub Discussions page
<https://github.com/PlasmaPy/PlasmaPy/discussions>`__ is a great place to
```

**Review by rocco8773 — CHANGES_REQUESTED** [review]
Just a few minor syntax issues.

**Comment by rocco8773** [comment]
There a few other places that need to be updated too...

1. `codemeta.json`
2. `.github/ISSUE_TEMPLATE/Bug_report.md`
   * It looks like riot is still referenced here too.
3. `.github/ISSUE_TEMPLATE/Feature_request.md`
   * It looks like riot is still referenced here too.

File: docs/COMMUNICATION.rst (hunk 1 above)
Comment (StanczakDominik) [reply docs/COMMUNICATION.rst]: Gah, sorry, i didn't check if the tests passed, just fired the pr off from terminal and forgot about it!

**Comment by StanczakDominik** [comment]
Should have been more careful with my grepping! Will clean this up tomorrow.

**Comment by rocco8773** [comment]
> Should have been more careful with my grepping! Will clean this up tomorrow.

Cool.  Just re-request my review when ready and I'll do the final once over before merging.

**Comment by StanczakDominik** [comment]
Looks like I did update the codemeta.json entry!

**Comment by Tiger-Du** [comment]
This PR is relevant to Issue #938 by the way. The PR template would need to be updated as well to solve that issue.

**Review by rocco8773 — APPROVED** [review]
LGTM!  Thank you for doing this.

**Pull request merged**